src/test/groovy/com/github/vkorobkov/jfixtures/JFixturesTest.groovy

```diff
@@ -85,12 +85,12 @@ class JFixturesTest extends Specification implements YamlVirtualFolder {
 
     def "MSSQL fixture to a string"() {
         expect:
-        JFixtures.msSql(tmpFolderPath as String).asString() == MSSQL_EXPECTED_SQL
+        JFixtures.microsoftSql(tmpFolderPath as String).asString() == MSSQL_EXPECTED_SQL
```

Comment:
Can we rename `MSSQL_EXPECTED_SQL` as well?

Reply:
If you insist - this is just a constant in our tests, not a public api. But yeah, I will do. Thanks ❤️